docs/getting-started/getting-started-with-docker.rst

```diff
@@ -22,15 +24,143 @@ There is a good chance that you have heard of containerization. A container is e
+When a user runs a Docker image locally, the Docker Engine spins up an instance of the Docker image. This instance is called a Docker container. You can think of an image as a template, and a container as something made from that template. Note that the filesystem of a Docker container is sequestered from the rest of your computer, so you will either want the container to already have any extra files you might need, or get those files into the Docker container by "mounting" them. We will briefly talking about how to mount files in this tutorial, but keep in mind that workflow languages handle this for us automatically, as will be shown in :doc:`Getting Started with Nextflow <getting-started-with-nextflow>`, :doc:`Getting Started with CWL <getting-started-with-cwl>`, and :doc:`Getting Started with WDL <getting-started-with-wdl>`. But we don't always need to mount files: when we talk about how to create a Dockerfile, you will learn how to create a Docker image that already has the files you need -- whether it's a simple 3 line Python script saved as a text file, or a complicated compiled program with lots of prerequisites.
+
+Docker is a robust piece of software with dozens of features. We won't be going over everything in this tutorial, but we will give you the overview needed to use Docker in the context of tools and workflows.
 
 What do I need to run Docker?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Docker generally recommends that people `install it via Docker Desktop <https://docs.docker.com/desktop/#download-and-install>`__. Docker Desktop includes the command line Docker Engine program that we will be using in this tutorial, plus a GUI to make adjustment of certain settings a little easier. You could instead `install the Docker Engine from binaries <https://docs.docker.com/engine/install/binaries/>`__, but this is generally not recommended as it is a little harder to install than Docker Desktop and does not automatically receive updates. In either case, to install a modern version of Docker, you will need a 64 bit system. Generally speaking you will also need root permissions on whatever system you are running on.
+Docker generally recommends that people `install it via Docker Desktop <https://docs.docker.com/desktop/#download-and-install>`__. Docker Desktop includes the command line Docker Engine program that we will be using in this tutorial, plus a GUI to make adjustment of certain settings a little easier, and a quick tutorial for new users (which we recommend you try out!). You could instead `install the Docker Engine from binaries <https://docs.docker.com/engine/install/binaries/>`__, but this is generally not recommended as it is a little harder to install than Docker Desktop and does not automatically receive updates. In either case, to install a modern version of Docker, you will need a 64 bit system. Generally speaking you will also need root permissions on whatever system you are running on.
```

Comment:
Could you link directly to the suggested tutorial?

Reply:
I can't, it's in the Docker Desktop program itself.